assets/src/components/ladderPage.tsx

```diff
@@ -71,8 +68,13 @@ const LadderPage = (): ReactElement<HTMLDivElement> => {
   )
 
   const vehicleAndRouteForNotification = useVehicleAndRouteForNotification(
-    selectedTripIdsForNotification
+    selectedNotification
   )
+  useEffect(() => {
+    if (vehicleAndRouteForNotification === null) {
+      dispatch(setNotificationIsInactive())
```

Comment:
Not necessary if you want to get this out a bit faster, but what if we showed the modal right from `ladderPage` instead of setting the state and drawing from `App`? It's all absolutely/fixedly positioned, so I think it'd work. (If you do this, you don't need to wait for another review on it)

Reply:
Personally I like having all the modals in one place; this is something about which reasonable people might disagree. But in my experience modals are one of those common elements that are surprisingly tricky to get right, so having them all co-located might make them easier to reason about.